### DISCUSSION TITLE
RWGltf_CafWriter - binary: false

### DISCUSSION THREAD

**Discussion by freddida** [donalffons/opencascade.js#166] [Q&A]
Hey all,

I am using the `RWGltf_CafWriter` to export gltf files.
When setting binary to `true` the glb file works perfectly.

I couldn't find an option to define the .bin file when setting binary to `false`.

Is there a possbility to export the file as .gltf + .bin?

Thanks in advance!

**Comment by donalffons** [comment]
I don't think `gltf+bin` is supported by OCCT. Therefore, it's not supported in ocjs either. However, this might be possible using [glTF-Transform](https://gltf-transform.donmccurdy.com/) using the `partition` feature, for example.
